src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
```

Comment:
The `add` method should not accept two parameters since it is supposed to use the `result` property of the calculator object. It should only take one parameter and add it to the `result`.